### ISSUE TITLE
Namespace Error Upon Opening

### ISSUE THREAD

**Issue by shirleycreates** [dmbfm/unity-ar-gps-location-mapbox-route-sample#1]
I'm having an issue as soon as I open the project. Getting several namespace errors. I tried to fix them in Visual Studio using the suggestions offered by Intellisense but it would break something else. I'm using Unity 2020.3.11f1. 

Any advice with this? 

![namespace_errors](https://user-images.githubusercontent.com/80176192/149672058-9c5cc940-34ce-4986-9f57-d36f96f68289.PNG)

**Comment by dmbfm** [comment]
Have you imported the AR+GPS Location asset into the project as per instructions?

**Comment by shirleycreates** [comment]
> 

Ah thanks for the quick response! I did but just realized I forgot to update it in the package manager 😅 I'm a Unity Newb ☺

**Comment by shirleycreates** [comment]
I was able to deploy it to my device but I'm getting a black screen in the AR camera. I'm using Android and have installed both AR core & AR foundation. I've also switched to ARM 64, is there something else I may be missing?

**Comment by Sayoj** [comment]
@shirleycreates have you activated ARCore in Edit>Project Settings...>XR Plug-In Management
![Capture](https://user-images.githubusercontent.com/73247844/149802806-1051a732-67bb-42f9-9ff1-a039a0269643.PNG)
?

**Comment by shirleycreates** [comment]
> @shirleycreates have you activated ARCore in Edit>Project Settings...>XR Plug-In Management ![Capture](https://user-images.githubusercontent.com/73247844/149802806-1051a732-67bb-42f9-9ff1-a039a0269643.PNG) ?

Yes, I figured it out shortly after :D It's working pretty well although I noticed the object that I placed at a specific coordinate was moving with me. It didn't stay stuck at its coordinate but I will continue playing with it. 

I'm also curious if I'm able to add any interactivity with the path renderer? Like a collider. I ask this because I'd like the user to follow a path and collect objects along the way (let's say coins for instance) but they would need to follow along the GPS coordinates to create a line from the beginning to the destination. 

Sorry, not sure if I should make a new thread for this >_<

**Comment by AnkitRaj0** [comment]
my project isnt opening !!! it says "failed to open, project not valid... !!"

**Comment by dmbfm** [comment]
> my project isnt opening !!! it says "failed to open, project not valid... !!"
I can't reproduce this issue at all... are you using Unity hub?

**Comment by dmbfm** [comment]
> I'm also curious if I'm able to add any interactivity with the path renderer? Like a collider. I ask this because I'd like the user to follow a path and collect objects along the way (let's say coins for instance) but they would need to follow along the GPS coordinates to create a line from the beginning to the destination.

In this case isn't it enough to have colliders on the coins?

**Comment by AnkitRaj0** [comment]
> > my project isnt opening !!! it says "failed to open, project not valid... !!"
> > I can't reproduce this issue at all... are you using Unity hub?



YES . THANKS FOR THE RESPONSE BUT I HAVE SOLVED IT ALREADY : )
